src/constants.ts

```diff
@@ -15,13 +15,16 @@ export enum runtimes {
     dotnetcore = 'dotnetcore',
     ruby = 'ruby',
     tomcat = 'tomcat',
-    javase = 'java|8-jre8'
+    javase = 'java|8-jre8',
+    python = 'python'
 }
 
 export function getIgnoredFoldersForDeployment(runtime: string): string[] {
     switch (runtime) {
         case runtimes.node:
             return ['node_modules{,/**}'];
+        case runtimes.python:
```

Comment:
I still think users name a virtual environment whatever they want. Ideally we're smarter about this. For example, you can "detect" a venv by check the path for an activate script. Something similar to https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/commands/createNewProject/PythonProjectCreator.ts#L200

That being said, I'm fine if you hard-code the most common "envs" here and file an issue to be smarter later

Reply:
Yeah, fair enough.  I'll add the environments that the .gitignore lists for now and make a smarter solution in a separate PR.